### ISSUE TITLE
Enabling the mode prints a message, even if used non-interactively

### ISSUE THREAD

**Issue by wasamasa** [lastquestion/explain-pause-mode#82]
This is more of a stylistic nitpick. Enabling a few dozen minor modes in my init file doesn't print a single message, but for explain-pause-mode it does. You could argue that for a diagnostic tool this is fine, but it's still a bit weird.

**Comment by lastquestion** [comment]
I agree with you, I just never made the effort to check if it was in `init.el` non-interactive and not display. 

Actually I've been thinking about it, and I probably will change it to display a message "if it fails to install", and be silent otherwise. This way makes more sense for both interactive and non-interactive.
